src/ui/utils/recording.ts

```diff
@@ -15,9 +16,23 @@ export function getRecordingURL(recording: Recording): string {
     id = slugify(recording.title, { strict: true }).toLowerCase() + SLUG_SEPARATOR + recording.id;
   }
 
+  // Revert once we merge the test-suites branch into main (see SCS-1723)
+  if (window.location.host === "tests.replay.io") {
+    return `https://app.replay.io/recording/${id}`;
+  }
+
   return `/recording/${id}`;
 }
 
+export function useGetRecordingURLForTest(recordingId: string): string {
```

Comment:
Will this mean that users will get "stuck" on app.replay.io once they open a recording? E.g. the in app back button will keep them on app.replay.io

Reply:
That's a good point. 

If this is a problem in the future we might need to use a NextJS middleware to redirect requests for the library when you're viewing a test workspace. Hopefully this is temporary though and we dont need it.

Reply:
@ryanjduffy still ok to just merge so we stop breaking telemetry?

Reply:
also probably good to do since test-suites will not have the latest and greatest